feg/gateway/services/n7_n40_proxy/servicers/session_controller.go

```diff
@@ -94,6 +95,22 @@ func (srv *CentralSessionController) TerminateSession(
 	ctx context.Context,
 	request *protos.SessionTerminateRequest,
 ) (*protos.SessionTerminateResponse, error) {
-
-	return (&protos.UnimplementedCentralSessionControllerServer{}).TerminateSession(ctx, request)
+	glog.V(2).Info("Deleting session")
+	if err := validateSessionTerminateRequest(request); err != nil {
+		return nil, err
```

Comment:
in those cases I personally like to print the error here 

```
if err := validateSessionTerminateRequest(request); err != nil {
		err = fmt.Errorf("Could not validate SessionTerminateRequest:" , err)
		glog.Error(err)
		return nil, err
```

The reason is, if you don't return an error, you will not really see an error on the feg log. The error will only be seen by the caller at AGW. 

Note I am not saying to print all the errors. But in this case this error is not printed anywhere in n7. 

See how all s8_proxy final errors on grpc have a log [here](https://sourcegraph.com/-/editor?remote_url=git%40github.com%3Amagma%2Fmagma.git&branch=master&file=feg%2Fgateway%2Fservices%2Fs8_proxy%2Fservicers%2Fs8_proxy.go&editor=JetBrains&version=v1.2.1&utm_product_name=IntelliJ+IDEA&utm_product_version=2021.2.2&start_row=76&start_col=25&end_row=76&end_col=25)

Reply:
Done